crates/eigen-cli/src/lib.rs

```diff
@@ -3,23 +3,32 @@
     issue_tracker_base_url = "https://github.com/Layr-Labs/eigensdk-rs/issues/"
 )]
 pub mod args;
+pub mod bls;
 mod convert;
 pub mod eigen_address;
 mod generate;
 mod operator_id;
+
 use eth_keystore::KeystoreError;
 use tokio::runtime::Runtime;
 
 use crate::eigen_address::ContractAddresses;
 use alloy_contract::Error as ContractError;
 use alloy_json_rpc::RpcError;
 use alloy_transport::TransportErrorKind;
-use args::{Commands, EigenKeyCommand};
+use args::{Commands, EigenKeyCommand, MnemonicLanguage};
 use ark_serialize::SerializationError;
+use bls::BlsKeystore;
+use colored::*;
 use convert::store;
 use eigen_crypto_bls::error::BlsError;
 use generate::KeyGenerator;
 use operator_id::derive_operator_id;
+use rust_bls_bn254::{
+    errors::KeystoreError as BlsKeystoreError, mnemonics::Mnemonic, CHINESE_SIMPLIFIED_WORD_LIST,
+    CHINESE_TRADITIONAL_WORD_LIST, CZECH_WORD_LIST, ENGLISH_WORD_LIST, ITALIAN_WORD_LIST,
+    KOREAN_WORD_LIST, PORTUGUESE_WORD_LIST, SPANISH_WORD_LIST,
+};
 use thiserror::Error;
 
 pub const ANVIL_RPC_URL: &str = "http://localhost:8545";
```

Comment:
Probably wanna make this configurable ?

Reply:
done. Added user can send the path to the word list , if not , we use ours